sign-page/src/App.css

```diff
@@ -0,0 +1,96 @@
+.head{
+   padding-right: 180px;
+    line-height: 63.68px;
+    
+}
+.form{
+    text-align: center;
+    width: 322px;
+    height: 340px;
+  
+}
+h3 {
+    font-weight: normal;
+    margin-left: 15px;
+}
+#email{
+    width:288px;
+    height: 33px;
+    display: flex;
+    margin-top: 8px;
+    margin-left: 32px;
+    border-radius: 4px;
```

Comment:
This css is repeated.
Instead use a class and set the classname to both the inputs.